polygeist/tools/cgeist/Lib/CodeGenTypes.cc

```diff
@@ -234,6 +234,31 @@ const CodeGenOptions &CodeGenTypes::getCodeGenOpts() const {
   return CGM.getCodeGenOpts();
 }
 
+static llvm::raw_ostream &
```

Comment:
Nice! I would place this static helper function at the top where we added a section on "Helper Functions".